hass-addon-sunsynk-dev/run.py

```diff
@@ -165,6 +203,21 @@ def startup() -> None:
 
         SENSORS.append(getfilter(fstr, sensor=sen))
 
+        if isinstance(sen, NumberRWSensor):
+            if isinstance(sen.min, Sensor):
+                sens_dependencies[sen.min.id] = sen.min
+            if isinstance(sen.max, Sensor):
+                sens_dependencies[sen.max.id] = sen.max
+
+    # Add any sensors that are depended upon which were not specified in OPT.sensors
+    for name, sen in sens_dependencies.items():
+        if name not in sens and sen != RATED_POWER:  # Rated power does not change
+            fstr = suggested_filter(sen)
+            msg[f"*{fstr}"].append(name)  # type: ignore
+            SENSORS.append(getfilter(fstr, sensor=sen))
+
+            _LOGGER.info("Added sensor %s as other sensors depend on it", name)
+
```

Comment:
Can't we rather define the min&max in the configuration if its different than the default in the definition?

Is there any case where the min/max could actually change?

Reply:
Scratch the previous comment. Looking at the code, the main issue will be that you cannot guarantee BATTERY_LOW_CAPACITY will be read **before** you do sensor discovery. 

Once you've done sensor discovery changing the battery low capacity will not update the discovered elements (I'm not sure we want to do this).

Maybe we can add RATED_POWER and BATTERY_LOW CAPACITY to be read on startup, so define a STARTUP_SENSORS.

STARTUP_SENSORS = (
SERIAL, RATED_CAPACITY, BATTERY_LOW_CAPACITY
)